extensions/MESA/MESA_framebuffer_flip_y.txt

```diff
@@ -59,7 +59,10 @@ Issues
 
 New Procedures and Functions
 
-    None
+    If ARB_framebuffer_no_attachments is not implemented then implement
```

Comment:
For ease of use, this extension should implement the below functions independently of the availability of `GL_ARB_framebuffer_no_attachments`. As written now, if a driver currently does not have `GL_ARB_framebuffer_no_attachments`, then the driver must provide the MESA functions; later, if the driver begins roviding `GL_ARB_framebuffer_no_attachments`, then the driver must *stop* providing the MESA functions, potentially breaking apps that use the MESA functions.

Reply:
Hmm... I have more opinions now :-)

The extension requires GL 4.3 or GLES 3.0. GL 4.3 already provides glFramebufferParameteri. So, you could reasonably change the symbol visibility aspect of the extension to:

  * In GL >= 4.3, no MESA functions. Only core functions.
  * In GL < 4.3, don't care, not supported.
  * In GLES >= 3.0. Always provide MESA functions.
  * In GLES >= 3.1. Support the core functions too.

Whether you choose to follow my first comment (provide MESA functions everywhere) or this comment (provide MESA functions in GLES only), I don't have a strong opinion.

Reply:
I think the second option makes sense.